lang/en.js

```diff
@@ -65,6 +65,7 @@ export default {
 	"components.input-time-range.errorBadInput": "{startLabel} must be before {endLabel}",
 	"components.input-time-range.startTime": "Start Time",
 	"components.interactive.instructions": "Press Enter to interact, Escape to exit",
+	"components.list.toolbar": "List toolbar",
```

Comment:
In the lms, you can create a langterm entry in the `definitions` file and then when translations go through it will create translations for all of the other languages that are missing it. Does it work differently in core where we need to manually create an entry for every language?

Reply:
So I actually used `npx mfv add-missing -s en -p ./lang/` to do this for me, because otherwise I _think_ `npm run test:translations` in CI will fail